sd_card/storage.c

```diff
@@ -56,6 +57,27 @@ void sd_write(void * p_event_data, uint16_t event_size)
 		{
 			NRF_LOG_INFO("Audio write to sd failed: %d", ff_result);
 		}
+
+    if (!audio_metadata_file_handle.err)
+    {
+      uint32_t seconds;
+      uint16_t milliseconds;
+      systick_get_timestamp(&seconds, &milliseconds);
```

Comment:
I think the you might be introducing a delay here because it's happening after the `f_write`. In the old version the tag `v0.0.0` of the code, the timestamp was done before it.